pkg/ride/functions_proto_test.go

```diff
@@ -736,7 +737,30 @@ func TestAssetInfoV4(t *testing.T) {
 }
 
 func TestBlockInfoByHeight(t *testing.T) {
-	t.SkipNow()
+	gen := newTestAccount(t, "GENERATOR")
+	rewards := proto.Rewards{proto.NewReward(gen.address(), 12345)}
+	bi := protobufBlockBuilder().withHeight(2).withGenerator(gen).withRewards(rewards)
+	env := newTestEnv(t).withLibVersion(ast.LibV6).withBlock(bi.toBlockInfo())
+	obj := blockInfoToObject(bi.toBlockInfo(), ast.LibV6)
+	for _, test := range []struct {
+		te   *testEnv
+		args []rideType
+		fail bool
+		r    rideType
+	}{
+		{env, []rideType{rideInt(0)}, false, rideUnit{}},
+		{env, []rideType{rideInt(-1)}, false, rideUnit{}},
+		{env, []rideType{rideInt(2)}, false, obj},
+		{env, []rideType{rideInt(3)}, true, nil},
```

Comment:
@alexeykiselev Is it valid that error will be returned instead of unit for blocks in **future** (non non existent blocks)?

Reply:
No, it won't, it is just a mock behaviour to test the error handling. In real life in this case the errNotFound will be returned and handled by returning Unit from function.